core/src/global.css

```diff
@@ -60,6 +60,10 @@
 	/* Shadow */
 	--shadow-1: 0px 1px 4px rgba(0, 0, 0, 0.1);
 	--shadow-2: 0px 1px 4px rgba(0, 0, 0, 0.1);
+
+	/* z-index, depend on each other */
+	--z-icon: 2;
+	--z-th: 3; /* th always > icon */
```

Comment:
This is good! It's great to have a centralized z-index control. It's CSS variables after all, so the consumers may change these to suit there needs (e.g. to increase all of Moai's z-index to be higher than them). However, this is not a recommended usage.

Reply:
This PR only covers 2 cases, but I think this should be applied for the whole project. You can do it here, or just create an issue so we don't forget

Reply:
yeah, I think We need to declare CSS variable for elements depends on each other.  We easily control, don't forget if something changes.